src/main/java/driver/SingletonDriver.java

```diff
@@ -1,12 +1,23 @@
 package driver;
 
+import io.github.bonigarcia.wdm.WebDriverManager;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
 
 public class SingletonDriver {
-    private static final WebDriver driver = new ChromeDriver();
+    static WebDriver driver;
+//    static WebDriver driver = new ChromeDriver();
+
+//    static SingletonDriver singletonDriver = new SingletonDriver();
+
+    private SingletonDriver() {
+    }
 
     public static WebDriver getDriver() {
+       if (null == driver) {
+           WebDriverManager.chromedriver().setup();
```

Comment:
please move this and the next line to the SingletonDriver constructor and call this constructor from here (so settings are inside of constructor)